examples/progress.js

```diff
@@ -0,0 +1,49 @@
+/* @jsx h */
+const {h, mount, Component, Text, Bar} = require('../');
+
+const MAX = 20;
+
+class ProgressApp extends Component {
+	constructor() {
+		super();
+
+		this.state = {
+			i: 0
+		};
+	}
+
+	render() {
+		const text = `Running `;
```

Comment:
Very good idea, I'd want that to be implemented natively too. Although I've been thinking and didn't come up with a way to solve this properly. Opened https://github.com/vadimdemedes/ink/issues/5 to discuss this.